File: app/src/main/java/com/suzei/minote/utils/Turing.java
package com.suzei.minote.utils;

import se.simbio.encryption.Encryption;

public class Turing {

    public static String encrypt(String textToEncrypt) {
        Encryption encryption = Encryption.getDefault(
                "MiNoteEncryptKey",
                "MiNoteSalt",
                new byte[16]);

        return encryption.encryptOrNull(textToEncrypt);
    }

    public static String decrypt(String textToDecrypt) {
        Encryption encryption = Encryption.getDefault(
                "MiNoteEncryptKey",
                "MiNoteSalt",
                new byte[16]);

        return encryption.decryptOrNull(textToDecrypt);
    }

}

File: app/src/main/java/com/suzei/minote/utils/Turing.kt
package com.suzei.minote.utils

import se.simbio.encryption.Encryption

object Turing {

    fun encrypt(textToEncrypt: String): String {
        val encryption = Encryption.getDefault(
                "MiNoteEncryptKey",
                "MiNoteSalt",
                ByteArray(16))

        return encryption.encryptOrNull(textToEncrypt)
    }

    fun decrypt(textToDecrypt: String): String {
        val encryption = Encryption.getDefault(
                "MiNoteEncryptKey",
                "MiNoteSalt",
                ByteArray(16))

        return encryption.decryptOrNull(textToDecrypt)
    }

}
